src/transformers/modeling_utils.py

```diff
@@ -30,6 +30,7 @@
 from typing import Any, Callable, Dict, List, Optional, Tuple, Union
 
 import torch
+from accelerate.hooks import AlignDevicesHook
```

Comment:
We need to keep this under an `is_accelerate_available()` check

Reply:
Thanks for that! I could not decipher the original error message, and have added the check